added adjustMovementSpeed function

plally opened the pull request.

plally committed:
added adjustMovementSpeed function

plally committed:
Update sv_pvp_movespeed.lua

plally committed:
Update sv_pvp_movespeed.lua

plally committed:
fixed indentation issue

overlordbleck commented on lua/autorun/server/sv_pvp_movespeed.lua:
```diff
@@ -29,11 +29,30 @@ local function playerIsInBuild( ply )
+	end
+    multiplier = movementMultiplier(wepCount) 
+    ply:SetRunSpeed(400*multiplier)
+    ply:SetWalkSpeed(200*multiplier) 
```
Is it possible to get the default run and walk values from a function

overlordbleck commented on lua/autorun/server/sv_pvp_movespeed.lua:
```diff
@@ -29,11 +29,30 @@ local function playerIsInBuild( ply )
+	for k, weapon in pairs(weapons) do
+		if nonEffectedWeapons[weapon:GetClass()] == nil then
+            wepCount =  wepCount+1
+            
```
I really think we should have a separate function so that the code is neater.

plally replied:
as in a separate function for the weapon count?

overlordbleck replied:
Yeah

plally committed:
walk and run speed constants, changed tabs to spaces

plally committed:
Update sv_pvp_movespeed.lua

plally committed:
fixed mistakes

overlordbleck commented:
Is this working?

plally commented:
the movementMultiplier function is still just a placeholder, i didnt fix the issue with that math

plally committed:
add setSpeed function, add functionality to movementMultiplier

copied generateCFCHook into sv_pvp_movespeed.lua
rpg and frag were added to nonEffectedWeapons because they are default weapons

plally commented:
the movementMultiplier function could still use some updating, i just made it functional
